goclean.sh

```diff
@@ -8,7 +8,7 @@
 set -ex
 
 # Automatic checks
-test -z "$(gometalinter --disable-all --enable=gofmt $(glide novendor) | tee /dev/stderr)"
+test -z "$(gometalinter --disable-all --enable=gofmt --enable gosimple --deadline=4m $(glide novendor) | tee /dev/stderr)"
```

Comment:
so i'm ok with this after all since it errors if it takes longer than the deadline.  it's not a silent error.